machiavelli-cards/pom.xml

```diff
@@ -14,19 +14,51 @@
     <description>machiavelli-cards</description>
 
     <properties>
+        <lombok.version>1.18.24</lombok.version>
+        <hikari.version>5.0.1</hikari.version>
     </properties>
 
     <dependencies>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.tomcat</groupId>
+                    <artifactId>tomcat-jdbc</artifactId>
```

Comment:
why is this excluded?

Reply:
My false. When i read article about hikari configuration i was thinking that i need to exclude tomcat-jdbc to let spring boot find hikari config. But this is another option of implementation. https://www.baeldung.com/spring-boot-hikari